### ISSUE TITLE
alert box when leaving page in Chrome - "Do you want to leave this site? Changes you made may not be saved."

### ISSUE THREAD

**Issue by andosteinmetz** [andosteinmetz/twohoursaweek#27]
@andrewdeutsch I think Chrome interprets something in the new button as user input, and asks if you want to lose your changes when navigating away. Is this happening for you as well?

**Comment by andosteinmetz** [comment]
No, I can't replicate that. It's working fine for me in Version 56.0.2924.87 (64-bit). The only change made was the target in the donate-button.php file.

**Comment by andrewdeutsch** [comment]
@andosteinmetz , no, I can't replicate that. It's working fine for me in Version 56.0.2924.87 (64-bit). The only change made was the target in the donate-button.php file.

**Comment by andosteinmetz** [comment]
Oh, weird... Maybe I've mis-diagnosed. When I leave the homepage, I get a alert dialog asking: "Are you sure you want to leave? Your changes may be lost." or something like that.

**Comment by andrewdeutsch** [comment]
I see that. Clicking anything activates it.

**Comment by andosteinmetz** [comment]
Turns out it's the mailchimp form. My bad. It must be a recent change to Chrome that's causing it, since the mailchimp form hasn't changed since the site launched.
